.github/workflows/documentation-website.lighthouse-check.yml

```diff
@@ -38,17 +36,14 @@ jobs:
         languages: javascript-typescript
         queries: security-and-quality
         source-root: documentation-website
-    - name: Setup Node.js environment ${{ env.CURRENT_NODE_VERSION }}
-      uses: actions/setup-node@v4
-      with:
-        node-version: ${{ env.CURRENT_NODE_VERSION }}
-        cache: 'npm'
-        cache-dependency-path: |
-          framework/package-lock.json
-          documentation-website/package-lock.json
-          history-microservice/package-lock.json
-          history-website/package-lock.json
-          cli/package-lock.json
-          package-lock.json
-    - run: cd documentation-website && npm install
-    - run: cd documentation-website && npm run lighthouse
+    - name: Install Nginx
+      run: sudo apt-get update && sudo apt-get install -y nginx
+    - name: Configure Nginx
+      run: |
+        # Replace the default Nginx configuration with your own
+        sudo cp ./nginx.conf /etc/nginx/sites-available/default
+        sudo systemctl restart nginx
+    - name: Run Lighthouse Check
+      run: |
+        npm ci
+        npm run lighthouse
```

Comment:
this script has not yet been adapted, it will still start a server of it's own

Reply:
What should I do here?

Reply:
adjust the script

Reply:
I have checked the package file, and there is `"lighthouse": "npm run build && start-server-and-test serve http://localhost:8080 'node tools/lighthouse-report.js'"` Isn't it adapted yet, and if not, which script is compatible now?

Reply:
the start server part has to be removed, only building and running the tools/lighthouse-report can stay.